tests/manage/z_cluster/test_must_gather.py

```diff
@@ -90,12 +89,12 @@ def test_must_gather(self):
         logger.info("Collecting logs - Done!")
 
         # Compare running pods list to "/pods" subdirectories
-        must_gather_helper = re.compile(r'must-gather-.*.-helper')
+        must_gather_helper = re.compile(r"must-gather-.*.-helper")
         logger.info("Checking logs tree")
         logs = [
-            logs for logs in self.get_log_directories(directory) if not (
-                must_gather_helper.match(logs)
-            )
+            logs
```

Comment:
not sure why this line spilt happened

Reply:
Probably because of how black separates sections of the list comprehension. I'm not going to fight the formatter on this if it is just going to change it back.